pkg/restapi/operation/didexchange/models/invitation.go

```diff
@@ -24,6 +24,31 @@ type GenericError struct {
 	} `json:"body"`
 }
 
+// CreateInvitationRequest model
+//
+// This is used for operation to create invitation
+//
+// swagger:parameters createInvitation
+type CreateInvitationRequest struct {
+	// Params for creating invitation
+	//
+	// in: path
+	*CreateInvitationParams
+}
+
+// CreateInvitationParams model
+//
+// This is used for creating invitation
+//
+type CreateInvitationParams struct {
+
+	// The Alias to be used in invitation to be created
+	Alias string `json:"alias"`
+
+	// The Public flag to create an invitation from the public DID
+	Public bool `json:"public,string,omitempty"`
```

Comment:
are these parameters in aca py?

Reply:
Yes, they have one more field called “auto” to auto accept requests for this connection. We don’t have that feature yet so I didn’t add this